src/components/Layer2ProductCard.tsx

```diff
@@ -110,21 +41,61 @@ const Layer2ProductCard: React.FC<IProps> = ({
   const intl = useIntl()
 
   return (
-    <Card>
-      <ImageWrapper background={background}>
-        <Image image={image} alt={alt} objectFit="contain" />
-      </ImageWrapper>
-      <Content className="hover">
-        <div>
-          <Title>{name}</Title>
-          {children && <Children>{children}</Children>}
-          <Description>{description}</Description>
+    <Flex
+      color="text"
+      boxShadow={"0px 14px 66px rgba(0, 0, 0, 0.07)"}
+      direction="column"
+      justify="space-between"
+      bg="searchBackground"
+      borderRadius={1}
+      border={"1px solid lightBorder"}
+      textDecoration="none"
+      padding={2}
+      _hover={{ transition: "transform 0.1s", transform: "scale(1.02)" }}
+    >
+      <Flex
+        bg={background}
+        direction="row"
+        justify="center"
+        align="center"
+        boxShadow="inset 0px -1px 0px rgba(0, 0, 0, 0.1)"
+        minH="200px"
+      >
+        <Image
+          as={GatsbyImage}
+          image={image}
+          alt={alt}
+          objectFit="contain"
+          alignSelf="center"
+          maxW={{ base: "311px", sm: "372px" }}
+          maxH={"257px"}
+        />
+      </Flex>
+      <Flex
+        p={6}
+        textAlign="left"
+        h="100%"
+        direction="column"
+        className="hover"
+      >
+        <Box>
+          <Heading as="h3" mt="0" mb={3}>
```

Comment:
![image](https://user-images.githubusercontent.com/468158/192845915-72d68d22-d22f-4850-9cc9-48ad3e9a4364.png)

this heading seems too big comparing it with the prod version

Reply:
So it seems chakra sizing for` h3 "lg"` is bigger than the sizing y'all are using. I messed around with the font size and used "24px" but when the screen size resolution goes to phone the font is smaller than what is being used in the prod version. The closest I got to it looking to the prod version is using `size="md"`.

Reply:
Oh ok, good point. Lets use `3xl` instead of `md` that is slightly smaller than the one we were using here (which was `2rem`).

Reply:
Hi are we sure we want to use `3xl` here? This will just make the heading even more bigger than what it originally had (`lg`)? Is so I will happily do so just making sure this is what you going for based on styling.

Here is a screenshot to showcase the heading with `3xl` instead of `md`
![image](https://user-images.githubusercontent.com/61770010/193730708-b8b22461-82ad-4041-aac8-73961786362d.png)

Reply:
So, on the preview deploy I see this:
![image](https://user-images.githubusercontent.com/468158/194175909-371c0fd3-29ee-4e70-be1b-80420ff65337.png)

and this is production:
![image](https://user-images.githubusercontent.com/468158/194175938-3c2df15b-485a-4cad-b0b2-53b9a80a3814.png)

Production is using 24px and the preview 20px.
24px = 1.5rem = 2xl in the chakra scale